src/swiftlet/azext_swiftlet/generated/_help.py

```diff
@@ -19,12 +19,16 @@
 
 helps['swiftlet virtual-machine list'] = """
     type: command
-    short-summary: "List all of the virtual machines in the specified subscription. Use the nextLink property in the \
-response to get the next page of virtual machines."
+    short-summary: "List all of the virtual machines in the specified resource group. Use the nextLink property in the \
+response to get the next page of virtual machines. And List all of the virtual machines in the specified subscription. \
+Use the nextLink property in the response to get the next page of virtual machines."
```

Comment:
Simplify with `List all of the virtual machines in the specified resource group or subscription.`?